lms/urls.py

```diff
@@ -237,6 +237,19 @@
     )
 
 if settings.COURSEWARE_ENABLED:
+    COURSE_URLS = patterns(
```

Comment:
What about all of the other URL declarations? I understand the time crunch, but this is a relatively low-impact change that can clear away some debt.